data/data/powervs/main.tf

```diff
@@ -4,21 +4,60 @@ provider "ibm" {
   zone             = var.ibmcloud_zone
 }
 
+data "ibm_resource_group" "cos_group" {
+  name = var.powervs_resource_group_name
+}
+
+resource "ibm_resource_instance" "cos_instance" {
+  name              = var.powervs_cos_instance_name
+  resource_group_id = data.ibm_resource_group.cos_group.id
+  service           = "cloud-object-storage"
+  plan              = "standard"
+  location          = var.powervs_cos_instance_location
+}
```

Comment:
I don't see a tag mentioned for this resource

Reply:
Good catch, need to tag this for cleanup. Done.